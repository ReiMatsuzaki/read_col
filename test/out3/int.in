produced by make_input.rb
1.0000000000000 0.0000000000 0 0
  1  2  5 15  6 65 65
  8  1SGG  1SGU  1PXU  1PXG  1PYU  1PYG  1B1G  1AU
  7
  2  3  4
  2  5  6
  2  7  8
  3  5  7
  3  6  8
  4  5  8
  4  6  7
  2  1  2                /AO ReDuction Sets 1
  1  1                   /AO ReDuction Sets 2
  1  2                   /AO ReDuction Sets 3 
  6  1  2  3  4  5  6    /AO ReDuction Sets 4
  4  3  4  5  6          /AO ReDuction Sets 5
  2  2  1                /set of SO Coef. 1
  1  1                     1SGG
  1 -1                     1SGU
  1  1  2                /set of SO Coef. 2
  1  			   1SGG
  1  6  2                /set of SO Coef. 3
 -1 -1  4  0  0  0	   1SGG
  1  2  3                /set of SO Coef. 4
  1 -1			   1SGU
  6  6  4                /set of SO Coef. 5
  0  0  1  0  0 -1	   1SGG
  0  0  1  0  0  1	   1SGU
  1  0  0  1  0  0	   1PXU
  1  0  0 -1  0  0	   1PXG
  0  1  0  0  1  0	   1PYY
  0  1  0  0 -1  0	   1PYG
  4  6  5                /set of SO Coef. 6
  1  0  0  1  0  0	   1PXU
  1  0  0 -1  0  0	   1PXG
  0  1  0  0  1  0	   1PYU
  0  1  0  0 -1  0	   1PYG
3, 1
13.360000, 0.000000, 0.032828, 0.000000
2.013000, 0.000000, 0.231204, 0.000000
0.453800, 0.000000, 0.817226, 0.000000
1, 1
0.123300, 0.000000, 1.000000, 0.000000
1, 1
0.041100, 0.000000, 1.000000, 0.000000
1, 1
0.013700, 0.000000, 1.000000, 0.000000
1, 2
0.800000, 0.000000, 1.000000, 0.000000
1, 2
0.200000, 0.000000, 1.000000, 0.000000
1, 2
0.050000, 0.000000, 1.000000, 0.000000
6, 1
0.132447, -0.027494, 0.111530, 0.068994
0.321839, -0.044423, 0.424320, 0.068000
0.827900, -0.067667, 0.384700, -0.054349
2.408084, -0.101692, 0.168100, -0.056960
8.604437, -0.158477, 0.047387, -0.021326
46.924645, -0.291453, 0.008619, -0.004317
6, 1
0.198670, -0.041241, 0.111530, 0.068994
0.482759, -0.066635, 0.424320, 0.068000
1.241851, -0.101500, 0.384700, -0.054349
3.612126, -0.152538, 0.168100, -0.056960
12.906656, -0.237716, 0.047387, -0.021326
70.386967, -0.437179, 0.008619, -0.004317
6, 1
0.088298, -0.018329, 0.111530, 0.068994
0.214559, -0.029615, 0.424320, 0.068000
0.551934, -0.045111, 0.384700, -0.054349
1.605389, -0.067795, 0.168100, -0.056960
5.736292, -0.105651, 0.047387, -0.021326
31.283097, -0.194302, 0.008619, -0.004317
6, 3
0.082532, -0.017475, 0.064550, 0.063625
0.160346, -0.025536, 0.386250, 0.117140
0.317166, -0.035899, 0.456970, -0.049078
0.672986, -0.050202, 0.207160, -0.091297
1.621016, -0.072164, 0.047817, -0.034783
5.051732, -0.115606, 0.005781, -0.005524
6, 3
0.123798, -0.026213, 0.064550, 0.063625
0.240520, -0.038304, 0.386250, 0.117140
0.475749, -0.053849, 0.456970, -0.049078
1.009479, -0.075303, 0.207160, -0.091297
2.431524, -0.108246, 0.047817, -0.034783
7.577597, -0.173409, 0.005781, -0.005524
6, 3
0.185697, -0.039319, 0.064550, 0.063625
0.360780, -0.057455, 0.386250, 0.117140
0.713623, -0.080773, 0.456970, -0.049078
1.514219, -0.112955, 0.207160, -0.091297
3.647286, -0.162370, 0.047817, -0.034783
11.366396, -0.260114, 0.005781, -0.005524
6, 3
0.055021, -0.011650, 0.064550, 0.063625
0.106898, -0.017024, 0.386250, 0.117140
0.211444, -0.023933, 0.456970, -0.049078
0.448657, -0.033468, 0.207160, -0.091297
1.080677, -0.048110, 0.047817, -0.034783
3.367821, -0.077071, 0.005781, -0.005524
6, 3
0.036681, -0.007767, 0.064550, 0.063625
0.071265, -0.011349, 0.386250, 0.117140
0.140963, -0.015955, 0.456970, -0.049078
0.299105, -0.022312, 0.207160, -0.091297
0.720452, -0.032073, 0.047817, -0.034783
2.245214, -0.051381, 0.005781, -0.005524
  H 7  2 1.0
0.00000         0.00000         0.70000
0.00000         0.00000         -0.70000
  2  1
 1 1
 2 1   s-1SGG 1SGU           
 3 4   s-1SGU                
 4 4   s-1SGU                
 5 5   p-1SGG 1SGU ...       
 6 6   p-1PXU 1PXG 1PYU 1PYG 
 7 6   p-1PXU 1PXG 1PYU 1PYG 
CEN 8  1 0.0
0.00000         0.00000         0.00000
 8 2    s-1SGG 
 9 2    s-1SGG 
 10 2   s-1SGG
 11 3   d-1SGG 
 12 3   d-1SGG 
 13 3   d-1SGG 
 14 3   d-1SGG 
 15 3   d-1SGG 
