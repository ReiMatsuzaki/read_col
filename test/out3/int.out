1produced by make_input.rb                                                       
 CSCALE =    1.00000     THETAD =   0.00000   NOTWO =   0  NEGL =   0  IRSTRT =   0  ITLIM =   0  NOLIM =   0
   1  2  5 15  6 65 65  0
   8  1SGG  1SGU  1PXU  1PXG  1PYU  1PYG  1B1G  1AU 
   7
   2  3  4
   2  5  6
   2  7  8
   3  5  7
   3  6  8
   4  5  8
   4  6  7
   2  1  2
   1  1
   1  2
   6  1  2  3  4  5  6
   4  3  4  5  6
   2  2  1
   1  1
   1 -1
   1  1  2
   1
   1  6  2
  -1 -1  4  0  0  0
   1  2  3
   1 -1
   6  6  4
   0  0  1  0  0 -1
   0  0  1  0  0  1
   1  0  0  1  0  0
   1  0  0 -1  0  0
   0  1  0  0  1  0
   0  1  0  0 -1  0
   4  6  5
   1  0  0  1  0  0
   1  0  0 -1  0  0
   0  1  0  0  1  0
   0  1  0  0 -1  0
   3  1    
    13.360000       0.0000000      0.32828000E-01   0.0000000    
    2.0130000       0.0000000      0.23120400       0.0000000    
   0.45380000       0.0000000      0.81722600       0.0000000    
   1  1    
   0.12330000       0.0000000       1.0000000       0.0000000    
   1  1    
   0.41100000E-01   0.0000000       1.0000000       0.0000000    
   1  1    
   0.13700000E-01   0.0000000       1.0000000       0.0000000    
   1  2    
   0.80000000       0.0000000       1.0000000       0.0000000    
   1  2    
   0.20000000       0.0000000       1.0000000       0.0000000    
   1  2    
   0.50000000E-01   0.0000000       1.0000000       0.0000000    
   6  1    
   0.13244700     -0.27494000E-01  0.11153000      0.68994000E-01
   0.32183900     -0.44423000E-01  0.42432000      0.68000000E-01
   0.82790000     -0.67667000E-01  0.38470000     -0.54349000E-01
    2.4080840     -0.10169200      0.16810000     -0.56960000E-01
    8.6044370     -0.15847700      0.47387000E-01 -0.21326000E-01
    46.924645     -0.29145300      0.86190000E-02 -0.43170000E-02
   6  1    
   0.19867000     -0.41241000E-01  0.11153000      0.68994000E-01
   0.48275900     -0.66635000E-01  0.42432000      0.68000000E-01
    1.2418510     -0.10150000      0.38470000     -0.54349000E-01
    3.6121260     -0.15253800      0.16810000     -0.56960000E-01
    12.906656     -0.23771600      0.47387000E-01 -0.21326000E-01
    70.386967     -0.43717900      0.86190000E-02 -0.43170000E-02
   6  1    
   0.88298000E-01 -0.18329000E-01  0.11153000      0.68994000E-01
   0.21455900     -0.29615000E-01  0.42432000      0.68000000E-01
   0.55193400     -0.45111000E-01  0.38470000     -0.54349000E-01
    1.6053890     -0.67795000E-01  0.16810000     -0.56960000E-01
    5.7362920     -0.10565100      0.47387000E-01 -0.21326000E-01
    31.283097     -0.19430200      0.86190000E-02 -0.43170000E-02
   6  3    
   0.82532000E-01 -0.17475000E-01  0.64550000E-01  0.63625000E-01
   0.16034600     -0.25536000E-01  0.38625000      0.11714000    
   0.31716600     -0.35899000E-01  0.45697000     -0.49078000E-01
   0.67298600     -0.50202000E-01  0.20716000     -0.91297000E-01
    1.6210160     -0.72164000E-01  0.47817000E-01 -0.34783000E-01
    5.0517320     -0.11560600      0.57810000E-02 -0.55240000E-02
   6  3    
   0.12379800     -0.26213000E-01  0.64550000E-01  0.63625000E-01
   0.24052000     -0.38304000E-01  0.38625000      0.11714000    
   0.47574900     -0.53849000E-01  0.45697000     -0.49078000E-01
    1.0094790     -0.75303000E-01  0.20716000     -0.91297000E-01
    2.4315240     -0.10824600      0.47817000E-01 -0.34783000E-01
    7.5775970     -0.17340900      0.57810000E-02 -0.55240000E-02
   6  3    
   0.18569700     -0.39319000E-01  0.64550000E-01  0.63625000E-01
   0.36078000     -0.57455000E-01  0.38625000      0.11714000    
   0.71362300     -0.80773000E-01  0.45697000     -0.49078000E-01
    1.5142190     -0.11295500      0.20716000     -0.91297000E-01
    3.6472860     -0.16237000      0.47817000E-01 -0.34783000E-01
    11.366396     -0.26011400      0.57810000E-02 -0.55240000E-02
   6  3    
   0.55021000E-01 -0.11650000E-01  0.64550000E-01  0.63625000E-01
   0.10689800     -0.17024000E-01  0.38625000      0.11714000    
   0.21144400     -0.23933000E-01  0.45697000     -0.49078000E-01
   0.44865700     -0.33468000E-01  0.20716000     -0.91297000E-01
    1.0806770     -0.48110000E-01  0.47817000E-01 -0.34783000E-01
    3.3678210     -0.77071000E-01  0.57810000E-02 -0.55240000E-02
   6  3    
   0.36681000E-01 -0.77670000E-02  0.64550000E-01  0.63625000E-01
   0.71265000E-01 -0.11349000E-01  0.38625000      0.11714000    
   0.14096300     -0.15955000E-01  0.45697000     -0.49078000E-01
   0.29910500     -0.22312000E-01  0.20716000     -0.91297000E-01
   0.72045200     -0.32073000E-01  0.47817000E-01 -0.34783000E-01
    2.2452140     -0.51381000E-01  0.57810000E-02 -0.55240000E-02
   H  7  2 1.
     0.00000000    0.00000000    0.70000000
     0.00000000    0.00000000   -0.70000000
   2  1
   1  1
   2  1
   3  4
   4  4
   5  5
   6  6
   7  6
 CEN  8  1 0.
     0.00000000    0.00000000    0.00000000
   8  2
   9  2
  10  2
  11  3
  12  3
  13  3
  14  3
  15  3
1            THIS PROGRAM COMPUTES INTEGRALS OVER SYMMETRY ORBITALS.
              THE ATOMIC ORBITALS ARE CONTRACTED GAUSSIAN ORBITALS.
              THE POINT GROUPS ARE LIMITED TO D2H AND ITS SUBGROUPS.

                                   R. M. PITZER
                                  H. F. SCHAEFER



0REFERENCES 

      SYMMETRY ANALYSIS (EQUAL CONTRIBUTIONS),
      R. M. PITZER, J.CHEM. PHYS. 58, 3111 (1973)./
      AO INTEGRAL EVALUATION (HONDO),
      M. DUPUIS, J. RYS, AND H. F. KING, J. CHEM. PHYS. 65, 111(1976).,













             produced by make_input.rb                                                       




0IRREDUCIBLE REPRESENTATION   1       2       3       4       5       6       7       8
 DEGENERACY                   1       1       1       1       1       1       1       1
 LABEL                       SGG     SGU     PXU     PXG     PYU     PYG     B1G     AU 



0DIRECT PRODUCT TABLE
    (SGG) (SGG) = SGG
    (SGG) (SGU) = SGU
    (SGG) (PXU) = PXU
    (SGG) (PXG) = PXG
    (SGG) (PYU) = PYU
    (SGG) (PYG) = PYG
    (SGG) (B1G) = B1G
    (SGG) (AU ) = AU 
    (SGU) (SGG) = SGU
    (SGU) (SGU) = SGG
    (SGU) (PXU) = PXG
    (SGU) (PXG) = PXU
    (SGU) (PYU) = PYG
    (SGU) (PYG) = PYU
    (SGU) (B1G) = AU 
    (SGU) (AU ) = B1G
    (PXU) (SGG) = PXU
    (PXU) (SGU) = PXG
    (PXU) (PXU) = SGG
    (PXU) (PXG) = SGU
    (PXU) (PYU) = B1G
    (PXU) (PYG) = AU 
    (PXU) (B1G) = PYU
    (PXU) (AU ) = PYG
    (PXG) (SGG) = PXG
    (PXG) (SGU) = PXU
    (PXG) (PXU) = SGU
    (PXG) (PXG) = SGG
    (PXG) (PYU) = AU 
    (PXG) (PYG) = B1G
    (PXG) (B1G) = PYG
    (PXG) (AU ) = PYU
    (PYU) (SGG) = PYU
    (PYU) (SGU) = PYG
    (PYU) (PXU) = B1G
    (PYU) (PXG) = AU 
    (PYU) (PYU) = SGG
    (PYU) (PYG) = SGU
    (PYU) (B1G) = PXU
    (PYU) (AU ) = PXG
    (PYG) (SGG) = PYG
    (PYG) (SGU) = PYU
    (PYG) (PXU) = AU 
    (PYG) (PXG) = B1G
    (PYG) (PYU) = SGU
    (PYG) (PYG) = SGG
    (PYG) (B1G) = PXG
    (PYG) (AU ) = PXU
    (B1G) (SGG) = B1G
    (B1G) (SGU) = AU 
    (B1G) (PXU) = PYU
    (B1G) (PXG) = PYG
    (B1G) (PYU) = PXU
    (B1G) (PYG) = PXG
    (B1G) (B1G) = SGG
    (B1G) (AU ) = SGU
    (AU ) (SGG) = AU 
    (AU ) (SGU) = B1G
    (AU ) (PXU) = PYG
    (AU ) (PXG) = PYU
    (AU ) (PYU) = PXG
    (AU ) (PYG) = PXU
    (AU ) (B1G) = SGU
    (AU ) (AU ) = SGG






0NUCLEAR REPULSION ENERGY    0.71428571




0PRIMITIVE AO INTEGRALS NEGLECTED IF EXPONENTIAL FACTOR BELOW 10**(-65)
 CONTRACTED AO AND SO INTEGRALS NEGLECTED IF VALUE BELOW 10**(-65)
 SYMMETRY ORBITAL INTEGRALS WRITTEN ON UNIT  3
1              H ATOMS

0NUCLEAR CHARGE   1.00

0CENTER            X               Y               Z
   1             0.00000000      0.00000000      0.70000000
   2             0.00000000      0.00000000     -0.70000000

0OPERATOR      CENTER INTERCHANGES
  1(ID.)        1  2
  2(GEN.)       2  1


          1S ORBITALS
0ORBITAL EXPONENTS          13.36000       0.000000       2.013000       0.000000      0.4538000       0.000000    
 CONTRACTION COEFFICIENTS  3.2827991E-02   0.000000      0.2312039       0.000000      0.8172258       0.000000    

          SYMMETRY ORBITALS
 CTR, AO  1SGG1  1SGU1
  1,  1S  1.000  1.000
  2,  1S  1.000 -1.000


          1S ORBITALS
0ORBITAL EXPONENTS         0.1233000       0.000000    
 CONTRACTION COEFFICIENTS   1.000000       0.000000    

          SYMMETRY ORBITALS
 CTR, AO  2SGG1  2SGU1
  1,  1S  1.000  1.000
  2,  1S  1.000 -1.000


          1S ORBITALS
0ORBITAL EXPONENTS         4.1100000E-02   0.000000    
 CONTRACTION COEFFICIENTS   1.000000       0.000000    

          SYMMETRY ORBITALS
 CTR, AO  3SGU1
  1,  1S  1.000
  2,  1S -1.000


          1S ORBITALS
0ORBITAL EXPONENTS         1.3700000E-02   0.000000    
 CONTRACTION COEFFICIENTS   1.000000       0.000000    

          SYMMETRY ORBITALS
 CTR, AO  4SGU1
  1,  1S  1.000
  2,  1S -1.000


          2P ORBITALS
0ORBITAL EXPONENTS         0.8000000       0.000000    
 CONTRACTION COEFFICIENTS   1.000000       0.000000    

          SYMMETRY ORBITALS
 CTR, AO  3SGG1  5SGU1  1PXU1  1PXG1  1PYU1  1PYG1
  1,   X  0.000  0.000  1.000  1.000  0.000  0.000
  1,   Y  0.000  0.000  0.000  0.000  1.000  1.000
  1,   Z  1.000  1.000  0.000  0.000  0.000  0.000
  2,   X  0.000  0.000  1.000 -1.000  0.000  0.000
  2,   Y  0.000  0.000  0.000  0.000  1.000 -1.000
  2,   Z -1.000  1.000  0.000  0.000  0.000  0.000


          2P ORBITALS
0ORBITAL EXPONENTS         0.2000000       0.000000    
 CONTRACTION COEFFICIENTS   1.000000       0.000000    

          SYMMETRY ORBITALS
 CTR, AO  2PXU1  2PXG1  2PYU1  2PYG1
  1,   X  1.000  1.000  0.000  0.000
  1,   Y  0.000  0.000  1.000  1.000
  2,   X  1.000 -1.000  0.000  0.000
  2,   Y  0.000  0.000  1.000 -1.000


          2P ORBITALS
0ORBITAL EXPONENTS         5.0000000E-02   0.000000    
 CONTRACTION COEFFICIENTS   1.000000       0.000000    

          SYMMETRY ORBITALS
 CTR, AO  3PXU1  3PXG1  3PYU1  3PYG1
  1,   X  1.000  1.000  0.000  0.000
  1,   Y  0.000  0.000  1.000  1.000
  2,   X  1.000 -1.000  0.000  0.000
  2,   Y  0.000  0.000  1.000 -1.000
1            CEN ATOMS

0NUCLEAR CHARGE   0.00

0CENTER            X               Y               Z
   1             0.00000000      0.00000000      0.00000000


          1S ORBITALS
0ORBITAL EXPONENTS         0.1324470     -2.7494000E-02  0.3218390     -4.4423000E-02  0.8279000     -6.7667000E-02   2.408084    
 -0.1016920       8.604437     -0.1584770       46.92464     -0.2914530    
 CONTRACTION COEFFICIENTS  0.1115291      6.8993173E-02  0.4243163      6.7998592E-02  0.3846965     -5.4349240E-02  0.1680984    
 -5.6959808E-02  4.7386536E-02 -2.1325899E-02  8.6189149E-03 -4.3169777E-03

          SYMMETRY ORBITALS
 CTR, AO  4SGG1
  1,  1S  1.000


          1S ORBITALS
0ORBITAL EXPONENTS         0.1986700     -4.1241000E-02  0.4827590     -6.6635000E-02   1.241851     -0.1015000       3.612126    
 -0.1525380       12.90666     -0.2377160       70.38697     -0.4371790    
 CONTRACTION COEFFICIENTS  0.1115291      6.8993193E-02  0.4243163      6.7998662E-02  0.3846965     -5.4349180E-02  0.1680984    
 -5.6959783E-02  4.7386541E-02 -2.1325892E-02  8.6189158E-03 -4.3169764E-03

          SYMMETRY ORBITALS
 CTR, AO  5SGG1
  1,  1S  1.000


          1S ORBITALS
0ORBITAL EXPONENTS         8.8298000E-02 -1.8329000E-02  0.2145590     -2.9615000E-02  0.5519340     -4.5111000E-02   1.605389    
 -6.7795000E-02   5.736292     -0.1056510       31.28310     -0.1943020    
 CONTRACTION COEFFICIENTS  0.1115292      6.8993169E-02  0.4243164      6.7998537E-02  0.3846965     -5.4349313E-02  0.1680984    
 -5.6959847E-02  4.7386542E-02 -2.1325911E-02  8.6189158E-03 -4.3169799E-03

          SYMMETRY ORBITALS
 CTR, AO  6SGG1
  1,  1S  1.000


          3D ORBITALS
0ORBITAL EXPONENTS         8.2532000E-02 -1.7475000E-02  0.1603460     -2.5536000E-02  0.3171660     -3.5899000E-02  0.6729860    
 -5.0202000E-02   1.621016     -7.2164000E-02   5.051732     -0.1156060    
 CONTRACTION COEFFICIENTS  6.4550136E-02  6.3624984E-02  0.3862505      0.1171397      0.4569704     -4.9078588E-02  0.2071601    
 -9.1297332E-02  4.7817004E-02 -3.4783090E-02  5.7809989E-03 -5.5240121E-03

          SYMMETRY ORBITALS
 CTR, AO  7SGG1
  1,  XX -1.000
  1,  YY -1.000
  1,  ZZ  2.000


          3D ORBITALS
0ORBITAL EXPONENTS         0.1237980     -2.6213000E-02  0.2405200     -3.8304000E-02  0.4757490     -5.3849000E-02   1.009479    
 -7.5303000E-02   2.431524     -0.1082460       7.577597     -0.1734090    
 CONTRACTION COEFFICIENTS  6.4550100E-02  6.3624947E-02  0.3862503      0.1171396      0.4569701     -4.9078564E-02  0.2071600    
 -9.1297281E-02  4.7816976E-02 -3.4783070E-02  5.7809956E-03 -5.5240090E-03

          SYMMETRY ORBITALS
 CTR, AO  8SGG1
  1,  XX -1.000
  1,  YY -1.000
  1,  ZZ  2.000


          3D ORBITALS
0ORBITAL EXPONENTS         0.1856970     -3.9319000E-02  0.3607800     -5.7455000E-02  0.7136230     -8.0773000E-02   1.514219    
 -0.1129550       3.647286     -0.1623700       11.36640     -0.2601140    
 CONTRACTION COEFFICIENTS  6.4550134E-02  6.3624921E-02  0.3862504      0.1171394      0.4569701     -4.9078778E-02  0.2071599    
 -9.1297383E-02  4.7816964E-02 -3.4783095E-02  5.7809934E-03 -5.5240120E-03

          SYMMETRY ORBITALS
 CTR, AO  9SGG1
  1,  XX -1.000
  1,  YY -1.000
  1,  ZZ  2.000


          3D ORBITALS
0ORBITAL EXPONENTS         5.5021000E-02 -1.1650000E-02  0.1068980     -1.7024000E-02  0.2114440     -2.3933000E-02  0.4486570    
 -3.3468000E-02   1.080677     -4.8110000E-02   3.367821     -7.7071000E-02
 CONTRACTION COEFFICIENTS  6.4550094E-02  6.3624949E-02  0.3862503      0.1171396      0.4569701     -4.9078533E-02  0.2071600    
 -9.1297265E-02  4.7816977E-02 -3.4783066E-02  5.7809958E-03 -5.5240084E-03

          SYMMETRY ORBITALS
 CTR, AO 10SGG1
  1,  XX -1.000
  1,  YY -1.000
  1,  ZZ  2.000


          3D ORBITALS
0ORBITAL EXPONENTS         3.6681000E-02 -7.7670000E-03  7.1265000E-02 -1.1349000E-02  0.1409630     -1.5955000E-02  0.2991050    
 -2.2312000E-02  0.7204520     -3.2073000E-02   2.245214     -5.1381000E-02
 CONTRACTION COEFFICIENTS  6.4550129E-02  6.3624937E-02  0.3862504      0.1171395      0.4569702     -4.9078709E-02  0.2071600    
 -9.1297358E-02  4.7816973E-02 -3.4783090E-02  5.7809948E-03 -5.5240116E-03

          SYMMETRY ORBITALS
 CTR, AO 11SGG1
  1,  XX -1.000
  1,  YY -1.000
  1,  ZZ  2.000
 IST,ITYP(IST),NSO(IST)
 1SGG  11
 2SGU   5
 3PXU   3
 4PXG   3
 5PYU   3
 6PYG   3
 7B1G   0
 8AU    0
 ISO, MS(ISO), MNL(ISO)
           1      1      1
           2      1      1
           3      1      2
           4      2      1
           5      2      1
           6      2      1
           7      2      4
           8      2      4
           9      2      4
          10      2      4
          11      2      4
          12      1      1
          13      1      1
          14      1      1
          15      1      1
          16      1      2
          17      1      2
          18      1      2
          19      1      2
          20      1      2
          21      1      2
          22      1      2
          23      1      2
          24      1      2
          25      1      2
          26      1      2
          27      1      2
          28      1      2
1   CX   1977
 ICXST    120
                105 S INTEGRALS IN 1 BLOCKS
                105 T INTEGRALS IN 1 BLOCKS
                105 V INTEGRALS IN 1 BLOCKS
               13312 INTEGRALS IN    17 BLOCKS
